### PR TITLE
Add isSingleSlice chart utility function.

### PR DESCRIPTION
**Pull request by asvinb**
## Summary

Addresses issue #2991 

## Checklist

- [x] My code is tested and passes existing unit tests.
- [ ] My code has an appropriate set of unit tests which all pass.
- [ ] My code is backward-compatible with WordPress 4.7 and PHP 5.6.
- [ ] My code follows the [WordPress](https://make.wordpress.org/core/handbook/best-practices/coding-standards/) coding standards.
- [ ] My code has proper inline documentation.
- [x] I have added a QA Brief on the issue linked above.
- [x] I have signed the Contributor License Agreement (see <https://cla.developers.google.com/>).

### CODE REVIEW

**Comment by felixarntz** [comment]
@asvinb Please keep in mind to connect PRs to their respective issue in the future using the ZenHub UI below the PR description :)

File: assets/js/util/chart.js [hunk 1]
```diff
@@ -46,3 +46,26 @@ export const getChartDifferenceArrow = ( difference, options = {} ) => {
 		<ChangeArrow direction={ difference > 0 ? 'up' : 'down' } invertColor={ invertColor } />
 	);
 };
+
+/**
+ * Checks if there is a single row of data or one row is contributing 100% of the total for a given dimension.
+ *
+ * @since n.e.x.t
+ *
+ * @param {Object} report Report data object.
```
Comment (eugene-manuilov) [inline assets/js/util/chart.js:55]:
```suggestion
 * @param {Array.<Object>} report An array of report data objects.
```

**Review by eugene-manuilov — CHANGES_REQUESTED** [review]
All looks good to me. Have one minor suggestion (which I am going to apply) and this is good to go.

**Review by eugene-manuilov — APPROVED** [review]
Looks good now!

**Pull request merged**